src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -424,11 +422,11 @@ private static void AddRange(SymbolSet foundTypes, SymbolSet currentTypes, Func<
             else
             {
                 // For a source project, find the project that that type was defined in.
-                var sourceProject = solution.GetProject(type.ContainingAssembly, cancellationToken);
+                var sourceProject =
+                    solution.GetProject(type.ContainingAssembly, cancellationToken) ??
```

Comment:
When does GetProject return null in this case?

Reply:
a pure metadata-type.  and looking at this code, i don't need to handle that, since this is the 'non-metadata' codepath.  I'll revise in followup.